static/cluster-dns.yml

```diff
@@ -22,7 +22,6 @@ rules:
   - apiGroups:
       - ""
     resources:
-      - endpoints
```

Comment:
From v1.11.0, CoreDNS does not watch Endpoints.
https://github.com/coredns/coredns/pull/6147